parsec/core/fs/workspacefs/workspacefs.py

```diff
@@ -331,6 +332,7 @@ def get_version_lister(self):
             FSError
         """
         path = FsPath(path)
+        await self.transactions.file_resize(path, offset)
```

Comment:
A negative offset means [end of file](https://github.com/Scille/parsec-cloud/blob/abe8aa0ac64fc05e2f2164b34618c7a8a95b074a/parsec/core/fs/workspacefs/file_transactions.py#L29). In this case, no need to resize.

Reply:
C'est fixé.